edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Service/Startup.cs

```diff
@@ -2,11 +2,13 @@
 namespace Microsoft.Azure.Devices.Edge.Hub.Service
 {
     using System;
+    using System.Linq;
```

Comment:
Why do we need `System.Linq`?

Reply:
VS Code brought that in automatically along with Microsoft.AspNetCore.Http. Maybe because Append() is an extension method?